packages/db/postgres/migrations/009-migrate-backup-urls.sql

```diff
@@ -0,0 +1 @@
+UPDATE "upload" SET "backup_urls" = (SELECT json_object_agg(url, jsonb_build_object('url', url, 'inserted_at', inserted_at)) FROM backup WHERE "upload_id" = "upload"."id");
```

Comment:
We have all the backup urls currently in Backup table, so I suppose we want to copy them in the migration?

@hugomrdias did you create a migration script for this?

Reply:
The migration script should be enough I think?

`UPDATE "upload" SET "backup_urls" = (SELECT json_object_agg(url, jsonb_build_object('url', url, 'created', inserted_at)) FROM backup WHERE "upload_id" = "upload"."id")`

will build the JSON structure e.g. `{"https://backup/":  { url: "https://backup/", created: '2022-05-18 10:30:00+00:00' }}` from the Backup table using `upload_id`.

Reply:
Yeah, this was mostly a call out that we need to setup a maintenance window. I think this works, even though we should test it in staging first.

Reply:
the nft.storage migration script was this https://github.com/nftstorage/nft.storage/pull/1664#issuecomment-1082183567

Reply:
@vasco-santos, do you think I should port this script for web3.storage so we can batch the updates for the rollout?

Reply:
yes we should be able to migrate without downtime, ie. deploy to prod, start writing to the new column and run script to migrate from table to column.